disttask/framework/dispatcher/dispatcher.go

```diff
@@ -211,6 +214,9 @@ func (d *dispatcher) handleRunning() error {
 }
 
 func (d *dispatcher) updateTask(taskState string, newSubTasks []*proto.Subtask, retryTimes int) (err error) {
+	if !d.ownerManager.IsOwner() {
```

Comment:
what if the owner is dropped just after this check? There is always an interval between time of check to time of use. Can we add a version to transactional storage to detect it?

Reply:
Good point!